framework/console/Controller.php

```diff
@@ -296,6 +307,18 @@ public function options($actionID)
     }
 
     /**
+     * Returns option alias names.
```

Comment:
maybe we should be include @see options

Can you please suggest sometihing description?